changelogs/fragments/1038-fix-homebrew-and-homebrew-cask-package-validation

```diff
@@ -0,0 +1,3 @@
+bugfixes:
+- homebrew - fix package name validation for packages and casks containing hypen `-`
+  (https://github.com/ansible-collections/community.general/pull/1038)
```

Comment:
```suggestion
- homebrew - fix package name validation for packages and casks containing hypen ``-``
  (https://github.com/ansible-collections/community.general/issues/1037).
- homebrew_cask - fix package name validation for packages and casks containing hypen ``-``
  (https://github.com/ansible-collections/community.general/issues/1037).
```

Reply:
@MichaelWasher this is still open.

Reply:
I don't understand what is being asked sorry. You would prefer to 4 lines instance?

Reply:
I changed it to the 4 lines changelog just incase
https://github.com/ansible-collections/community.general/pull/1038/files#diff-961d0154e27fe6fd59ec4e79ec36fb5de7a0fa2689a5e8f92288fcbde3af2ee4R1-R5

Reply:
I'm also fine if you put it in two lines :) The problems with your changelog fragment were:
1. You only mentioned the homebrew module, not the homebrew_cask module.
2. You didn't use correct formatting for the dash (this is reStructuredText, not MarkDown).
3. A dot is missing at the end of the entry.